src/Model/CollectionInfoCommandIterator.php

```diff
@@ -41,6 +55,12 @@ class CollectionInfoCommandIterator extends IteratorIterator implements Collecti
      */
     public function current()
     {
-        return new CollectionInfo(parent::current());
+        $info = parent::current();
+
+        if ($this->databaseName !== null && isset($info['idIndex']) && ! isset($info['idIndex']['ns'])) {
+            $info['idIndex']['ns'] = $this->databaseName . '.' . $info['name'];
```

Comment:
Is this related to [PHPLIB-499](https://jira.mongodb.org/browse/PHPLIB-499) (#718)? If so, I'm curious if the spec change needs to be revised. https://github.com/mongodb/specifications/commit/8ed5d562f073f2e43e24c092492fa5ffe1a6b115 for [SPEC-1399](https://jira.mongodb.org/browse/SPEC-1399) only addressed the [`listIndexes`](https://docs.mongodb.com/manual/reference/command/listIndexes/), but I gather this pertains to some behavioral change in [`listCollections`](https://docs.mongodb.com/manual/reference/command/listCollections).

Reply:
Yes, which is why I missed this in the original PR. `listCollections` returns information about the `_id` index, which also omits the `ns` key in 4.4+. I've created [SPEC-1584](https://jira.mongodb.org/browse/SPEC-1584) to track this and will add this to the spec.